### DISCUSSION TITLE
does this number change? if not make it a timer

### DISCUSSION THREAD

**Discussion by Quantaviousdingleberry420** [aeolus-1/triangleGame.github.io#59] [Ideas]
<img width="632" alt="Screen Shot 2022-04-22 at 11 03 34 am" src="https://user-images.githubusercontent.com/104042540/164576528-f4cf87ed-c105-4210-8658-0e0b0cc79387.png">

**Comment by Bruce-Browninggg** [comment]
yes it does change. you finished that in 434 seconds
i am also a smart little teachers pet

**Reply by Quantaviousdingleberry420** [reply]
i didnt ask you clancy
